src/nvim/mbyte.c

```diff
@@ -1375,7 +1375,7 @@ static int dbcs_ptr2char(const char_u *p)
  */
 int utf_ptr2char(const char_u *p)
 {
-  int len;
+  uint8_t len;
```

Comment:
Since you're already doing it in other places, this could be moved down to the initialization.